stable/prometheus/values.yaml

```diff
@@ -332,13 +332,9 @@ server:
   ##
   image:
     repository: prom/prometheus
-    tag: v1.8.1
+    tag: v2.0.0
```

Comment:
Prometheus 2.1 has a lots of improvements. Can we bump to 2.1?

https://github.com/prometheus/prometheus/releases/tag/v2.1.0